test/integration/test_win_group1.yml

```diff
@@ -9,3 +9,4 @@
     - { role: win_fetch, tags: test_win_fetch }
     - { role: win_regmerge, tags: test_win_regmerge }
     - { role: win_regedit, tags: test_win_regedit }
+    - { role: win_acl_inheritance, tags: test_win_acl_inheritance }
```

Comment:
So this is not necessary, and even undesirable because it causes merge conflicts.